src/components/singleProductPage/SliderSection.jsx

```diff
@@ -42,27 +41,12 @@ const SINGLE_PRODUCT_SLIDER_SETTINGS = {
         slidesToScroll: 1,
       },
     },
-    {
-      breakpoint: 576,
-      settings: {
-        slidesToShow: 1,
-        slidesToScroll: 1,
-      },
-    },
-
-    {
-      breakpoint: 375,
-      settings: {
-        slidesToShow: 1,
-        slidesToScroll: 1,
-      },
-    },
   ],
 };
 
 export default function SliderSection() {
   return (
-    <Container className="singleProductContainer">
+    <Container>
```

Comment:
is there a reason this code is being changed here? if so, include it in the PR description. imagine from my perspective, i don't know if these are intentional changes or not

Reply:
I committed this class unintentionally, then I delete it

Reply:
pls update the PR description with a sentence about changes outside the scope of the title in that case, the reason is that if someone needs to go back and look at this PR they will immediately know from the PR descriptoin that there were additional changes.